src/Kaleidoscope-LEDEffect-Rainbow.h

```diff
@@ -9,6 +9,7 @@ class LEDRainbowEffect : public LEDMode {
   LEDRainbowEffect(void) {}
 
   void brightness(byte);
+  void delay(byte);
```

Comment:
Perhaps this should be called `update_delay` for consistency and clarity? `delay` alone tells little about what the setting is for.